### PR TITLE
clockwork cuirass now compatible with snakes. Now if only coiling was a mechanic we had

### PR DESCRIPTION
**Pull request by KeRSedChaplain**
### About The Pull Request
Adds naga taur versions of the clockwork cuirass, and makes the armor able to have other tauric variants added later.

### Why Its Good For The Game
Makes the whole sprite covered instead of just the top half

### Changelog
:cl:
Add: Naga taur mob sprite
Tweak: The cuirass can now support tauric bodies
/:cl:

### CODE REVIEW

**Comment by Ghommie** [comment]
Sadly, due to how garbage the current taur suits code is, I'm pretty sure you'll also have to sprite the other taur versions too or be cursed with invisible suit overlays issues.
Please standby until I convert tauric modes etcetera to flags and fix this design flaw. Geeze I hate this.

**Comment by KeRSedChaplain** [comment]
yeah thats fair. do I close this then?

**Comment by Ghommie** [comment]
No, keep it open, I'm making a big PR to fix several issues with these badly coded furry-related features right now.

**Comment by KeRSedChaplain** [comment]
closing this for the time being since I need to redo this

**Comment by KeRSedChaplain** [comment]
found out it will work normally so im gonna re-open this

**Comment by Ghommie** [comment]
> 
> 
> found out it will work normally so im gonna re-open this

Say that again after testing the clockwork cuirass on a mob with a deer taur accessory, I dare you.
Keep in mind #10112 will remove that awful universal tauric variable and replace it with taur type flags to help your laziness anway.

File: code/modules/antagonists/clockcult/clock_items/clockwork_armor.dm [hunk 1]
```diff
@@ -70,6 +70,7 @@
 	resistance_flags = FIRE_PROOF | ACID_PROOF
 	armor = list("melee" = 60, "bullet" = 70, "laser" = -25, "energy" = 0, "bomb" = 60, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 100)
 	allowed = list(/obj/item/clockwork, /obj/item/clothing/glasses/wraith_spectacles, /obj/item/clothing/glasses/judicial_visor, /obj/item/mmi/posibrain/soul_vessel, /obj/item/reagent_containers/food/drinks/bottle/holyoil)
+	tauric = TRUE		//Citadel Add for tauric hardsuits
```
Comment (Linzolle) [inline code/modules/antagonists/clockcult/clock_items/clockwork_armor.dm:73]: change to `mutantrace_variation = STYLE_DIGITIGRADE|STYLE_SNEK_TAURIC`
the tauric var is no longer used

**Pull request closed without merging**